Add the ability to specify not to send certificates with messages

drvespa opened the pull request:
* User reported issue to add ability to not send certificates with #7 
messages. Added new flag -ds to indicate this
* Add unit tests to cover the new scenarios that I added.

drvespa committed:
Add the ability to specify not to send certificates with messages

User reported issue to add ability to not send certificates with
messages. Added new flag -ds to indicate this
Add unit tests to cover the new scenarios that I added.

stephenegriffin commented on MapiSmimeToolkit.cpp:
```diff
@@ -140,7 +142,13 @@ BOOL ParseArgs(int argc, _TCHAR* argv[], ToolkitOptions * pToolkitOptions)
 			case 'd':
-				pToolkitOptions->bDefaultSecurityProfule = true;
+				if (_tcsclen(argv[i]) < 3) {
+					pToolkitOptions->bDefaultSecurityProfule = true;
```
Typo: bDefaultSecurityProfule => bDefaultSecurityProfile

stephenegriffin replied:
and really not a fan of just checking length here but I suppose for quick and dirty (very dirty) it kinda works

drvespa replied:
I fixed the typo and made the parsing of the new switch a little more definitive. I don't like it, but it'll do for now.

drvespa committed:
Replace profule with profile

drvespa committed:
Changing the parsing logic for the new switch

drvespa committed:
Add -d switch to set the default security profile test

Confirmed that my alteration of the handling of the 'd' switch didn't
cause issues with setting the default security profile.

drvespa committed:
Fixing Typo

RufusJWB commented:
Thank you for working on this issue! Do you have a ETA for this PR?

RufusJWB commented:
Did you make progress with this topic?

drvespa committed:
Add compile exclusions for test modules

Added compile exclusions for test modules
 MapiSmimeToolKit.cpp
 SecurityProfileUnitTests.cpp

drvespa committed:
Added missing switches as per the tester's comments
